arm/Microsoft.Network/applicationGateways/.bicep/nested_rbac.bicep

```diff
@@ -2,11 +2,10 @@ param roleAssignment object
 param builtInRoleNames object
 param resourceName string
 
-resource nested_rbac 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignment.principalIds: {
+resource roleAssigment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignment.principalIds: {
```

Comment:
Should be 

`resource nested_rbac 'Microsoft.Network/applicationGateways/providers/roleAssignments@2020-04-01-preview' = [for principalId in roleAssignment.principalIds: {`